### ISSUE TITLE
Missing icon on Linux

### ISSUE THREAD

**Issue by mikalehmann** [qTox/qTox#2530]
As you can see here ![](http://i.imgur.com/dbUZhPK.png), there is no qTox icon on Kubuntu.
Also, if you search for qTox in the application launcher, it doesn't show an icon.
I'm not sure if this also applies to other distributions as well.

**Comment by ovalseven8** [comment]
Not long ago I tested qTox on Kubuntu 15.10 (Plasma 5) with the build from the Tox website and there was no icon in the system tray as well.
But when I compiled qTox by myself, it worked. So, it's probably a problem concerning the packaging and static linking.

**Comment by agilob** [comment]
@zetok it's a packaging problem, the same on mint xfce, qtox doesn't have an icon, qtox is not accessible from application menu.

**Comment by zetok** [comment]
@agilob I no longer have an access to jenkins - someone who has it has to do it.

**Comment by Zer0-One** [comment]
This issue is blocked until #2599 is resolved

**Comment by ghost** [comment]
Maybe the title should be changed, because this issue does not only apply to Kubuntu, nor Ubuntu.

**Comment by dolohow** [comment]
I am fetching qtox on ubuntu from

```
deb http://pkg.tox.chat/debian nightly wily
```

and the icon is missing.

**Comment by ghost** [comment]
@dolohow, I have getting it from the same place, but now the icon is present.

**Comment by dolohow** [comment]
This is the version I have

```
Version: 1.3.0-28-ge37e87d-1
```

**Comment by ghost** [comment]
@dolohow: I seem to have the same version as you, have you tried reinstalling it? And how long has the issue been present? Maybe it's just stayed behind from an older version and all you need is a reinstall...

**Comment by dolohow** [comment]
I unpacked the package and I don't see any icon files in it

```
└── usr
    ├── bin
    │   └── qtox
    └── share
        ├── applications
        │   └── qTox.desktop
        ├── doc
        │   └── qtox
        │       ├── changelog.Debian.gz
        │       ├── copyright
        │       ├── README.Debian
        │       └── README.md
        └── menu
            └── qtox
```

qtox.desktop referes to icon file, but it is missing.

I also updated the icon cache, no luck.

**Comment by SMOKExRA** [comment]
The same problem on Ubuntu 15.10

**Comment by ByteCommander** [comment]
No indicator icon any more with qTox version v1.4.0-14-g5045c46 fetched as tar.gz-ed binary from GitHub on Ubuntu 16.04 as well.

**Comment by dolohow** [comment]
Actually I was going to report that the icon is no longer missing after my upgrade to 16.04 from 15.10

**Comment by zetok** [comment]
Community builds (https://github.com/tux3/qTox/blob/e797bf958279ea9eaa858bd377532b22de79bef7/INSTALL.md#community-builds) have it working on Debian 8 MATE, could you check if it works for you?

**Comment by zetok** [comment]
No response for over a month, assuming it was fixed, thus closing.

If issue is still not fixed, please comment for reopen.

**Comment by Nicole6** [comment]
Hi, I wanted to ask if this issue has been resolved. I just installed qTox following the github instructions, and seem to have exactly the problem described here: I cannot find a qTox icon that would open qTox when I click on it. Is such an icon available and I did something wrong on installation, or is it now only possible to open qTox from the command line?

**Comment by anthonybilinski** [comment]
@Nicole6 this issue should be resolved. What OS are you using qTox on and how did you install qTox?

**Comment by Nicole6** [comment]
Many thanks for your quick reply. I'm using Linux Ubuntu, and followed the instructions on this page: https://github.com/qTox/qTox/blob/master/INSTALL.md
I am a complete Linux beginner and have only very superficial understanding of what I did. At the end of the instructions I was able to open qTox by typing 
> ./qtox 

But no launching app seems to have been created, and I cannot find one when using a search function on my computer.
